client/src/app/site/motions/services/motion-multiselect.service.ts

```diff
@@ -247,4 +250,35 @@ export class MotionMultiselectService {
             }
         }
     }
+
+    /**
+     * Triggers the selected motions to be moved in the call-list (sort_parent, weight)
+     * as children or as following after a selected motion.
+     *
+     * @param motions The motions to move
+     */
+    public async bulkMoveItems(motions: ViewMotion[]): Promise<void> {
+        const title = this.translate.instant('This will assign the selected motions as belonging to:');
+        const options = ['Set as parent', 'Insert after'];
+        const allMotions = this.repo.getViewModelList();
+        const tree = this.treeService.makeTree(allMotions, 'weight', 'sort_parent_id');
+        const itemsToMove = this.treeService.getTopItemsFromTree(tree, motions);
+        const selectableItems = this.treeService.getTreeWithoutSelection(tree, motions);
+
+        const selectedChoice = await this.choiceService.open(title, selectableItems, false, options);
+        if (selectedChoice) {
+            if (selectedChoice.action === options[0]) {
```

Comment:
sounds like options[0] and options[1] could reference an enum.
Would improve readability to access `options[actions.whatever-zero-means-here]`

Reply:
"options" is only used inside this function, and declared ten lines earlier. I could check for the string itself (which would make it more readable, but may introduce errors if the choices are to be changed, if that helps readability. (If this is desired, I should change similar cases at user-list.component, too)

Reply:
OK. In case the functionality grows, we might want to put options in a struct/enum.
A comment above the declaration of the options array will be sufficient for now.